handsontable/src/editors/handsontableEditor/__tests__/a11y/navigation.spec.js

```diff
@@ -0,0 +1,122 @@
+describe('a11y navigation', () => {
```

Comment:
The same as https://github.com/handsontable/handsontable/pull/10535#discussion_r1357907159.